tests/cli/data/test_prop2po_files/stderr.txt

```diff
@@ -0,0 +1 @@
+root: WARNING: trying to parse ./data/test_prop2po_files/one.properties with encoding: iso-8859-1 but detected encoding is utf-8 (confidence: 1.0)
```

Comment:
That was intended, because in the code path the detected encoding is not used at all. However I have no clue what ever original intentions here...